<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarangoniTrackingDialog</class>
 <widget class="QDialog" name="MarangoniTrackingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracking</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Control options</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>63</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_PatternType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Point-wise spot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arc (+circle)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New light Pattern</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbox_OpMode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Identification</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Control FeedBack</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Control FeedForward</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Manual Mode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New Control Mode</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Identification Options</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Laser position</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_LasPartD">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Laser-Particle Dist. [mm] (Limit 1.1-5)</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_LasOrient">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Bottom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Top</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Y-Position</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="activateBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Turn ON mirror and Activate Processing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QSlider" name="horPosition">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>211</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="sliderPosition">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSlider" name="thresholdSlider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>205</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Target X-Position</string>
   </property>
  </widget>
  <widget class="QSlider" name="vertPosition">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>236</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="sliderPosition">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="threshEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>U(s)=Kp(E(s)+1/(Ti s) E(s)+Td s^2 E(s))</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>367</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>+ 1/(Tt s) (Usat(s)-U(s))</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ti</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Td</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>u sat [mm/s]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Windup Tt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_kp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Ti">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Td">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Usat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5.4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Tt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tsamp=1/FPS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Tsamp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0666</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>360</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>K ff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Kff">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tt=f( Tsamp,Ti)</string>
   </property>
  </widget>
  <widget class="QSlider" name="Xlaser_slider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>221</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QSlider" name="Ylaser_slider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>221</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x laser</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y laser</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Only in MANUAL MODE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkbox_Tracking">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate linear Tracking</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Xtarg">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Ytarg">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
